weave-js/src/components/Panel2/PanelNumber.tsx

```diff
@@ -1,16 +1,127 @@
-import {displayValueNoBarChart} from '@wandb/weave/common/util/runhelpers';
-import React from 'react';
+import React, {useCallback} from 'react';
 
 import * as CGReact from '../../react';
 import * as Panel2 from './panel';
 import {Panel2Loader} from './PanelComp';
+import * as ConfigPanel from './ConfigPanel';
+import {formatNumber} from '../../core/util/number';
+import {Tooltip} from '../Tooltip';
+import {IconHelpAlt} from '../Icon';
+import {MOON_500} from '@wandb/weave/common/css/color.styles';
 
 const inputType = {
   type: 'union' as const,
   members: ['none' as const, 'number' as const],
 };
 type PanelNumberProps = Panel2.PanelProps<typeof inputType>;
 
+const CustomFormatHelp = () => {
+  return (
+    <Tooltip
+      trigger={
+        <IconHelpAlt
+          color={MOON_500}
+          width={16}
+          height={16}
+          style={{margin: 4}}
+        />
+      }
+      content="Supports a subset of Python's Format Specification Mini-Language. e.g. Value 1234 with format '+010,.1f' -> '+001,234.0'"
```

Comment:
hmm what is missing? seems kind of spooky to say a subset and leaving it to the user to guess

Reply:
A bunch of things marked TODO in number.ts - some of it is pretty esoteric. I put in this little tooltip to make the field not completely unusable but a better solution would be to have full documentation and a link to that. Didn't want to invest time in that if it turned out there wasn't general buy-in on using Python's formats for custom. (E.g. Google Sheets has its own custom format strings which would be another approach.)

Agree that this text isn't good - I can turn it into a list of some things we do support and anything not mentioned will be a hidden feature.